--- origsrc/Text-BibTeX-0.80/inc/MyBuilder.pm	2017-03-25 21:41:12.000000000 +0100
+++ src/Text-BibTeX-0.80/inc/MyBuilder.pm	2017-04-01 15:56:23.777407000 +0200
@@ -45,7 +45,7 @@ sub ACTION_install {
  	}
 
     if ($^O =~ /cygwin/i) { # cygwin uses windows lib searching (PATH instead of LD_LIBRARY_PATH)
-        $self->install_path( 'usrlib' => '/usr/local/bin' );
+        $self->install_path( 'usrlib' => '/usr/bin' );
     }
     elsif (defined $self->{properties}{install_base}) {
         $usrlib = catdir($self->{properties}{install_base} => 'lib');
@@ -193,7 +193,7 @@ sub ACTION_create_objects {
         $object =~ s/\.c/.o/;
         next if $self->up_to_date($file, $object);
         $cbuilder->compile(object_file  => $object,
-                           extra_compiler_flags=>["-D_FORTIFY_SOURCE=1",@EXTRA_FLAGS],
+                           extra_compiler_flags=>[@EXTRA_FLAGS],
                            source       => $file,
                            include_dirs => ["btparse/src"]);
     }
