--- origsrc/IO-Socket-SSL-2.049/Makefile.PL	2017-04-27 20:43:14.000000000 +0200
+++ src/IO-Socket-SSL-2.049/Makefile.PL	2017-07-16 21:36:31.054203100 +0200
@@ -117,7 +117,7 @@ my %usable_ca;
     }
 }
 
-my $xt = $ENV{NO_NETWORK_TESTING} && 'n';
+my $xt = $ENV{NO_NETWORK_TESTING} ? 'n' : 'y';
 $xt ||= $yesno->( "Should I do external tests?\n".
     "These test will detect if there are network problems and fail soft,\n".
     "so please disable them only if you definitely don't want to have any\n".
